database/migrator.go

```diff
@@ -17,7 +17,7 @@ import (
 
 var migrationMutex sync.Mutex
 
-func RunMigrations(logger log.Logger, config DatabaseConfig) error {
+func RunMigrations(logger log.Logger, config DatabaseConfig, migrationsSource source.Driver) error {
```

Comment:
If we want to keep `base` package compatible with old versions of the service, I can create `RunMigrationsWithSource` to support `pkger` migrations as well.